### PR TITLE
Feature/adjust UI

### PR DESCRIPTION
**Pull request by godgiraffe**
# Fix UI issues

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -73,17 +77,19 @@ function calcRevolvingLoan() {
...
     }
 
     supplyEarning += theRoundSupplyEarning;
     brrowEarning += theRoundBrrowEarning;
 
     totalYieldValue = totalYieldValue + yieldValue;
-    console.log(`第 ${y} 輪, 拿 ${theRoundValue.toFixed(2)} 去押, 得到的利息為： ${yieldValue.toFixed(2)} , 目前總部位為： ${totalValue.toFixed(2)}, 總利息為：${totalYieldValue.toFixed(2)}`);
+    // console.log(`第 ${y} 輪, 拿 ${theRoundValue.toFixed(2)} 去押, 得到的利息為： ${yieldValue.toFixed(2)} , 目前總部位為： ${totalValue.toFixed(2)}, 總利息為：${totalYieldValue.toFixed(2)}`);
 
     const tbodyRef = document.getElementsByClassName("calcDetailContent")[0];
```
Comment (NorthBei) [inline js/app.js:94]: Why we don't use ```getElementById```?

File: js/app.js [hunk 2]
```diff
@@ -48,6 +50,8 @@ function calcRevolvingLoan() {
   const startValue = parseInt(document.getElementById("starting-value").value);
   // 循環借貸次數(Lending Loops)
   const loopNums = document.getElementById("lending-loops").value;
+  // 每次交易的手續費
+  const FeePerTraction = document.getElementById("fee-per-traction").value;
```
Comment (NorthBei) [inline js/app.js:54]: Why don't use camelcase as same as others variable name?

File: js/app.js (hunk 1 above)
Comment (godgiraffe) [reply js/app.js:94]: cuz, i don't know

**Review by NorthBei — APPROVED** [review]

**Review by CT77777 — APPROVED** [review]
ok

**Pull request merged**